.gitignore

```diff
@@ -1 +1,7 @@
-vendor/
+/vendor
+/.idea
+/.vscode
+/.vagrant
+.env
+/database/database.sqlite
+/.php_cs.cache
```

Comment:
newline at end please

Reply:
Whoops. We're going to find some.
Recently installed php-cs-fixer, so hope it will get better.

Reply:
This says "done" but it's still missing the newline